src/index.html

```diff
@@ -2,13 +2,58 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta name="viewport" content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
-    <link rel="stylesheet" href="./style.css">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <div class="wrapper">
+        <a class="logo-link" href="#">
+          <img
+            src="./images/logo.png"
+            alt="moyo-logo"
+            class="logo-img"
+          >
+        </a>
+        <nav class="header-nav">
+          <ul class="nav-list">
```

Comment:
```suggestion
          <ul class="nav__list">
```
fix all class names using `__`